boards/ct/ctcc/doc/index.rst

```diff
@@ -117,55 +174,113 @@ logs on emulated USB port.
 Debugging
 =========
 
-The ``ctcc/nrf52840`` board target does not have an on-board J-Link debug IC, however
+The ``ctcc`` boards does not have an on-board J-Link debug IC, however
 instructions from the :ref:`nordic_segger` page also apply to this board,
 with the additional step of connecting an external debugger.
 
-Option 2: Using MCUboot with DFU support
-----------------------------------------
+Option 2: Using MCUboot with mcumgr support
+-------------------------------------------
 
-It is also possible to use the MCUboot bootloader with DFU support to flash
-Zephyr applications. You need to flash MCUboot with DFU support and fill in slot0 with
-some application one-time using Option 1. Then you can re-flash an application using DFU utility
-by loading images to slot1. Note, it's not possible to have only MCUboot and load directly
-software to slot0 due to DFU implementation in Zephyr, which for present slot0 and slot1 in flash
-map, it assumes only slot1 partition as writeable.
+It is also possible to use the MCUboot bootloader with ``mcumgr`` support to flash
+Zephyr applications.
 
-Install ``dfu-util`` first and make sure MCUboot's ``imgtool`` is
+Install :ref:`mcu_mgr` first and make sure MCUboot's ``imgtool`` is
 available for signing your binary for MCUboot as described on :ref:`west-sign`.
 
-Next, do the **one-time setup** to flash MCUboot with DFU support.
-We'll assume you've cloned the `MCUboot`_ as a submodule when initializing
-Zephyr repositories using :ref:`west` tool.
+#. Compile MCUboot as a Zephyr application with ``mcumgr`` support.
```

Comment:
rather than any of this section, add a config file to https://github.com/mcu-tools/mcuboot for your board targets instead in the `boot/zephyr/boards` directory, then you can just build any application with sysbuild and include a working MCUboot in the build (or enable MCUboot by default if you so wish)

Reply:
I removed these `CONFIG_*`. I'll provide a `.conf` files to mcuboot later on